src/main/java/org/betonquest/betonquest/compatibility/protocollib/NPCGlow.java

```diff
@@ -229,29 +229,11 @@ public void run() {
         applyAll();
     }
 
-    /**
-     * Resets all glowing NPCs.
-     *
-     * @param profiles List of profiles that will get the packet
-     */
-    public void resetGlow(final Collection<? extends OnlineProfile> profiles) {
```

Comment:
Why you remove this? If quester remove the glow section in config, the npc will still glow

Reply:
I removed it because I just thought that the next update will be there in 5 ticks.  But you are right, if there isn't any config anymore the glow will persist...